source/extensions/filters/http/grpc_json_transcoder/json_transcoder_filter.cc

```diff
@@ -465,7 +465,7 @@ Http::FilterTrailersStatus JsonTranscoderFilter::encodeTrailers(Http::HeaderMap&
   response_in_.finish();
 
   const absl::optional<Grpc::Status::GrpcStatus> grpc_status =
-      Grpc::Common::getGrpcStatus(trailers);
+      Grpc::Common::getGrpcStatus(trailers, true);
```

Comment:
Would it make sense to add tests to the json_transcoder_filter_test for this change?